bi-connectors/PowerBIConnector/docs/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+All notable changes to the connector will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0.1] 2022-02-28
+### Added
+- Added option to allow users to choose a certificate validation option
```

Comment:
What do you think of this wording -- "Allow users to enable or disable validation for SSL certificate validation."?

Reply:
Sounds good to me